### PR TITLE
FreeCodeCamp: Responsive Web Design instruction update

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [[FreeCodeCamp Responsive Web Design Curriculum Update: Point to specific part of the legacy course.](https://www.notion.so/FreeCodeCamp-Responsive-Web-Design-Curriculum-Update-Point-to-specific-part-of-the-legacy-course-dae7a1b395e54fa58fae8ccce6baefdf)]

## Description:
Instructions updated to be more specific with the addition of a new freecodecamp responsive web design curriculum

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Sbonelo01 — APPROVED** [review]

File: content/topics/free-code-camp/responsive-web-design/4-applied-accessibility/_index.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
-Look under "Responsive Web Design"
+Look under "Legacy Responsive Web Design Certification".
 
 You'll be doing "Applied Accessibility"
 
 Please submit a link to your profile. For example, https://www.freecodecamp.org/fccfda32191-55c2-49ec-b6f4-26719c1dd7f4
 
-To make your profile useful, you need to go into your FreeCodeCamp settings and and set all your privacy settings to "Public".
\ No newline at end of file
+To make your profile useful, you need to go into your FreeCodeCamp settings and and set all your privacy settings to "Public".
```
Comment (Vuyisanani) [inline content/topics/free-code-camp/responsive-web-design/4-applied-accessibility/_index.md]: typo double  `and`

**Review by Vuyisanani — CHANGES_REQUESTED** [review]
Looking good, just one small typo, where you have 2 and in one sentence next to eat other

**Review by Sbonelo01 — APPROVED** [review]
Requested changes have been addressed

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]
This is a quick fix, when in doubt, try to stay away from legacy things

**Pull request merged**